File: tests/conformance/fixtures/go/10-task-inputs.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("build").Run("go build -o /out/app").
		Output("binary", "/out/app")

	p.Task("test").Run("./app test").
		InputFrom("build", "binary", "/app")

	p.Emit()
}

File: tests/conformance/fixtures/rust/10-task-inputs.rs
use sykli::Pipeline;

fn main() {
    let mut p = Pipeline::new();
    p.task("build").run("go build -o /out/app").output("binary", "/out/app");
    p.task("test").run("./app test").input_from("build", "binary", "/app");
    p.emit();
}
